aclk/agent_cloud_link.h

```diff
@@ -12,14 +12,18 @@
 #define ACLK_JSON_IN_TOPIC "callback-topic"
 #define ACLK_JSON_IN_URL "payload"
 
+//#define ACLK_SLEEP_LOOP 1   // query processing thread will sleep, thread wakeup disabled for now
 
 #define ACLK_MSG_TYPE_CHART "chart"
+#define ACLK_CHART_TOPIC "chart"
 #define ACLK_ALARMS_TOPIC "alarms"
 #define ACLK_METADATA_TOPIC "meta"
 #define ACLK_COMMAND_TOPIC "cmd"
 #define ACLK_TOPIC_STRUCTURE "/agent/%s"
 
-#define ACLK_STARTUP_WAIT 30             // Seconds to wait before establishing initialization process
+#define ACLK_DELAY_SEED 1.61803        // Exponential fibonnaci backoff
```

Comment:
It says in the spec that we will use powers of two.

Reply:
Will change to power of 2 to much the doc